defaults/main.yml

```diff
@@ -11,3 +11,5 @@ os_container_clusters_auth: {}
 # List of magnum container cluster templates. Each item should be a dict
 # mapping container cluster template attribute names to their values.
 os_container_clusters_templates: []
+
+old_ansible_python_interpreter: "{{ '/usr/bin/python' + ansible_facts.python.version.major | string }}"
```

Comment:
I don't think any other roles do it like this. That fact isn't necessarily what ansible_python_interpreter is set to. But you can't reference ansible_python_interpreter in defaults since it cause a loop.

See how it's done in os-images and os-flavors. You can specify the `vars` directly on the task here rather than using import_tasks, since there's only one task to apply it to.

Reply:
@markgoddard done.